### ISSUE TITLE
Monitor, page speed and infrastructure width

### ISSUE THREAD

**Issue by marcelluscaio** [bluewave-labs/checkmate#1252]
I believe we should apply ```{ width: "100%", gap: "1rem" }``` to all three main pages, so our aside will have exactly the size needed and no more, bringing better use of screen area

**Comment by gorkem-bwl** [comment]
@peterpardo do you want to work on this?

**Comment by peterpardo** [comment]
> @peterpardo do you want to work on this?

Yeah, sure! I'll knock this one out.

**Comment by gorkem-bwl** [comment]
> > I believe we should apply `{ width: "100%", gap: "1rem" }` to all three main pages, so our aside will have exactly the size needed and no more, bringing better use of screen area
> 
> Hello! The three remaining pages you were referring to are the `Monitors`, `Pagespeed`, and the `Infrastructure` pages?

Yep! Those are the main 'features' of Checkmate. 

@marcelluscaio are you talking about only those 3 pages, or all the sidebar pages (Incidents, Maintenance, Account, Other) should be included in this as well?

**Comment by marcelluscaio** [comment]
Yes, those 3 were the ones I had in mind when creating the Issue, but maybe we should discuss this to be sure of what we want before assigning that to someone

**Comment by gorkem-bwl** [comment]
> Yes, those 3 were the ones I had in mind when creating the Issue, but maybe we should discuss this to be sure of what we want before assigning that to someone

Sure. What was in your mind exactly? Just asking since there are more sidebar entries than those 3 links.

**Comment by gorkem-bwl** [comment]
@marcelluscaio ping here :)
